LambdaRuntimeDockerfiles/Infrastructure/src/Infrastructure/Infrastructure.csproj

```diff
@@ -9,11 +9,8 @@
 
   <ItemGroup>
     <!-- CDK Construct Library dependencies -->
-    <PackageReference Include="Amazon.CDK" Version="1.74.0" />
-    <PackageReference Include="Amazon.CDK.Pipelines" Version="1.74.0" />
-    <PackageReference Include="Amazon.CDK.AWS.CodePipeline" Version="1.74.0" />
-    <PackageReference Include="Amazon.CDK.AWS.CodeCommit" Version="1.74.0" />
-    <PackageReference Include="Amazon.CDK.AWS.CodePipeline.Actions" Version="1.74.0" />
+	<PackageReference Include="Amazon.CDK.Lib" Version="2.44.0" />
+	<PackageReference Include="Constructs" Version="[10.0.0,11.0.0)" />
```

Comment:
nit: Do we have to have a dependency on `Constructs`? Seems like that is brought in via `Amazon.CDK.Lib`.

Reply:
Removed